### PR TITLE
fixed some of the style issues

### PR DESCRIPTION
**Pull request by jmcampanini**
@zoubeiri @uschi2000

### CODE REVIEW

**Comment by uschi2000** [comment]
cool, thanks Javier! do other files need attention as well or is it just this one?

**Comment by jmcampanini** [comment]
not sure. tagging @zoubeiri for adding this to the weekly goals

File: README.md [hunk 1]
```diff
@@ -171,11 +171,11 @@ Oracle's Java style guide. The major changes we've made to Google's
 guide:
 
 - Substitute "Palantir" for "Google"
-- License or copyright information\_
-- Block indentation: +4 spaces\_
-- Column limit: 120\_
-- Import statements\_
-- Trailing Horizontal whitespace\_ is discouraged
+- License or copyright information
```
Comment (uschi2000) [inline README.md]: so these are no longer links?

**Comment by uschi2000** [comment]
any progress here?

**Comment by jmcampanini** [comment]
not yet. going to see if we can dedicate some time to baseline stuff this coming week :(

@zoubeiri for prioritizing SA

**Pull request merged**